### DISCUSSION TITLE
Bluetooth Low Energy Connection Subrating feature

### DISCUSSION THREAD

**Discussion by changshik** [zephyrproject-rtos/zephyr#58871] [Q&A] [answered]
Are there plans to have Connection Subrating feature in the BLE host stack? Have somebodies already been working on it?

**Comment by hermabe** [comment, marked as answer]
No current plans as far as I know. What is your usecase?

Subrating is a controller feature anyway, so not much host support would be needed.

**Reply by changshik** [reply]
My usecase is what the feature is aiming for, to not waste 6 CIs to transition from one state to another. I think it would be better to have host APIs for subrating feature like bt_conn_le_param_update() works for connection parameter update. Can anybody contribute to the implementation?

**Reply by hermabe** [reply]
Zephyr is an open source project, anyone can contribute. You can take a look at the [contribution guidelines](https://docs.zephyrproject.org/latest/contribute/index.html) for some guidelines for how to do it. The in-tree controller does not support the subrating feature, so testing is a bit more work but you can probably set up a unittest.
